network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/youtube_regrets_2021.html

```diff
@@ -15,11 +15,16 @@
 
 
 {% block content_wrapped %}
+{% with en_report="https://assets.mofoprod.net/network/documents/Mozilla_YouTube_Regrets_Report.pdf" de_report="https://assets.mofoprod.net/network/documents/Mozilla_YouTube_Regrets_Report_German.pdf" %}
```

Comment:
Let's land this for now, but make sure we replace this with code that taps into the actual env vars we have that make these URLs what they are (i.e. the AWS locations we use)